### PR TITLE
Fix homepage category URLs

### PR DESCRIPTION
**Pull request by elwoodxblues**
This PR removes hardcoded URLs to categories in homepage blocks.

### CODE REVIEW

File: templates/home.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
                 <div class="home--content-inner">
                   <h1>{% trans "Promo &amp; Sale" %}</h1>
                   <h2>{% trans "form the Northpole" %}</h2>
-                    <a class="btn btn-secondary home__button" href="/products/category/default-1/">{% trans "Shop now" %}</a>
+                    <a class="btn btn-secondary home__button" href="{% if featured_categories %}{{ featured_categories.0.get_absolute_url }}{% endif %}">{% trans "Shop now" %}</a>
```
Comment (bogdal) [inline templates/home.html]: I think `for` loop fits here better.

Comment (elwoodxblues) [inline templates/home.html]: Those homepage blocks aren't the same, if I used `for-loop` here I'd need to do custom checks whether to display full-width block or half-width.

**Comment by patrys** [comment]
This method of customization feels like a hack since there's no way to change anything but the links and I'm afraid even their order may change unpredictably.

**Comment by elwoodxblues** [comment]
I don't see better option for now, since we don't have any CMS that would allow to manage homepage.

**Comment by elwoodxblues** [comment]
@patrys Those blocks don't refer to any particular category at the moment so the order of URLs isn't that important. We'll come up with a better solution in the future, right now it allows us remove hardcoded URLs. If there are no other issues I'd like to merge it and release it soon as we have broken links in public demo.

**Comment by patrys** [comment]
Why not link them to `#`? I'm not sure we're improving things here, just making them broken in a slightly different way.

**Comment by elwoodxblues** [comment]
@mirekm wanted "Shop Now" buttons to link somewhere.

**Comment by patrys** [comment]
I believe #626 tracks a proper solution (allow banners to be managed in dashboard).

**Pull request merged**